metadata-ingestion/examples/transforms/example.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM 795586375822.dkr.ecr.us-west-2.amazonaws.com/datahub-executor:v0.3.8.2-acryl
```

Comment:
This should probably be a build arg: https://docs.docker.com/build/building/variables/

Reply:
This is an example. Nobody will be using this actually.

Reply:
Doesn't mean we can't have a good default for folks.

Reply:
If this is meant to be just an example. Please do explicitly call it out in `dataset_transfomer.md`

Reply:
It is in examples folder. That is explicit

Reply:
The whole heading https://datahubproject.io/docs/metadata-ingestion/docs/transformer/dataset_transformer/#writing-a-custom-transformer-from-scratch is an example already.